src/functionaltests/java/com/ericsson/ei/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -78,6 +78,10 @@
 
     private ObjectMapper objMapper;
 
+    final static private String entryPointQueryAggregatedObject = "/queryAggregatedObject";
+    final static private String entryPointQuery = "/query";
+    final static private String entryPointFailedNotifications = "/failed-notifications";
```

Comment:
I still want these tests to be moved from this class. Since the endpoints have no relation to each other, the tests should reflect this and also be restructured.

Reply:
It is out of scope of this story... you need to create a new story